homework-submission/barbarosapp/src/components/item.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import CheckBox from "./checkBox";
+
+class Item extends React.Component {
+  render() {
+    const { name, deadLine, handleCheck, done, id } = this.props;
+
+    return (
+      <section className="ToDos">
+        <div className="CheckBoxDiv">
+          <CheckBox handleCheck={handleCheck} id={id} done={done} />
+        </div>
+
+        <div className="Info">
+          <p>
+            Name : <span className="Color1"> {name} </span>
+          </p>
+          <p>
+            Deadline : <span className="Color2"> {deadLine} </span>
+          </p>
+        </div>
+
+        <div className="removeButtonDiv">
+          <button
+            onClick={item => this.handleRemove(item.id)}
```

Comment:
Here you have `onClick` event, my question is what is the value of `item.id` here?

Reply:
Yes, I corrected remove button problem. Thanks